src/app/page.tsx

```diff
@@ -7,7 +7,8 @@ import PageContainer from "@/components/PageContainer";
 import SearchBar from "@/components/Searchbar";
 import { useSearchParams } from "next/navigation";
 import bgBackground from "../public/bg-background.png";
```

Comment:
important: rename `bg-background.png` to `homepage-about-background.png`, for a cleaner and clearer name.

rename variable from `bgBackground` to `aboutBackground`.

Reply:
it's already been